src/scripts/main.js

```diff
@@ -1,3 +1,50 @@
 'use strict';
 
-// write code here
+const appendrow = document.querySelector('.append-row');
+const removerow = document.querySelector('.remove-row');
```

Comment:
```suggestion
const appendRow = document.querySelector('.append-row');
const removeRow = document.querySelector('.remove-row');
```
Don't forget about camelCase